helm/charts/determined/templates/master-config.yaml

```
@@ -70,6 +70,12 @@ data:
       slot_resource_requests:
         cpu: {{ .Values.slotResourceRequests.cpu }}
       {{- end }}
+      {{- if .Values.fluent }}
+      fluent:
+        {{- if .Values.fluent.image }}
+        image: {{ .Values.fluent.image }}
+        {{- end }}
```

could we pass some structures from values to here as-is?

```suggestion
        {{ toYaml .Values.fluent | nindent 8}}
```

---

probably. i just did what i knew would work. let me run it this way real fast.

---

i guess the difference is then the structure of `.Values.fluent` must match but condition that is implicit.

---

yeah, it works fine. maybe we can follow up to change it all. but i'll keep it this way since it matches the entire rest of the master config.

---

yes, matching helm and master config formats is the purpose.
- less mental load for a user because that config means the same in both places as it's taken literally
- no need for us to manually render these field-by-field in the helm template